dashboard/app/reporting_email.go

```diff
@@ -271,8 +272,19 @@ func emailReport(c context.Context, rep *dashapi.BugReport) error {
 	if err := mailTemplates.ExecuteTemplate(body, templ, rep); err != nil {
 		return fmt.Errorf("failed to execute %v template: %v", templ, err)
 	}
-	log.Infof(c, "sending email %q to %q", rep.Title, to)
-	return sendMailText(c, cfg, rep.Title, from, to, rep.ExtID, body.String())
+	title := generateEmailBugTitle(rep, cfg)
+	log.Infof(c, "sending email %q to %q", title, to)
+	return sendMailText(c, cfg, title, from, to, rep.ExtID, body.String())
+}
+
+func generateEmailBugTitle(rep *dashapi.BugReport, emailConfig *EmailConfig) string {
+	title := ""
+	if emailConfig.SubsystemsInSubject {
+		for i := len(rep.Subsystems) - 1; i >= 0; i-- {
+			title = fmt.Sprintf("[%s?] %s", rep.Subsystems[i].Name, title)
```

Comment:
Why there is "?" in the string?

Reply:
To underline that it's only a bot's suggestion, not anything definitive.